lib/index.js

```diff
@@ -39,12 +40,12 @@ flyd.stream = function(initialValue) {
   s.end = endStream;
   s.fnArgs = [];
   endStream.listeners.push(s);
-  s.toJSON = function() {
-    return s();
-  };
   if (arguments.length > 0) s(initialValue);
   return s;
 }
+// fantasy-land Applicative
+flyd.of = flyd.stream;
```

Comment:
Is this one necessary? It's not required for FL and `flyd.stream` does the same?

Reply:
good catch, will remove